astropy/modeling/tests/test_blackbody.py

```diff
@@ -73,6 +76,9 @@ def test_blackbody_scipy():
     np.testing.assert_allclose(intflux, ans.value, rtol=0.01)  # 1% accuracy
 
 
+@pytest.mark.filterwarnings("ignore:BlackBody provides the same capabilities")
+@pytest.mark.filterwarnings(
+    r"ignore:Input contains invalid wavelength/frequency value\(s\)")
```

Comment:
I'm not sure if this warning should be ignored or caught. Although only one of the values is raising it so may be it should have been a separate test.

Reply:
I'll look into this.

Reply:
@nden , I catch them like you suggested.